update documentation about the storagepath setting (#636)
FirelyTeam/firely-docs-firely-server: releasenotes/releasenotes_v6.rst

#. Evaluation of :ref:`Subscriptions<feature_subscription>` is now turned off by default. To enable - adjust ``SubscriptionEvaluatorOptions`` accordingly.
#. ``BundleOptions.DefaultTotal`` from now on has a default value of ``none``. For available options see :ref:`bundle_options`.

> #. ``TaskFileManagement.StoragePath`` was already marked as obsolete but had logic to still be used, this logic has now been removed. Use the ``TaskFileManagement.StorageService`` settings to provide the storage path, see :ref:`feature_bulkdataexport` for details.

Reviewer: ```suggestion
#. ``TaskFileManagement.StoragePath`` was already marked as obsolete, and is now also no longer forward compatible. Use the ``TaskFileManagement.StorageService`` settings to provide the storage path, see :ref:`feature_bulkdataexport` for details.
```

Reviewer: I found the original sentence a bit hard to read. How about this?
